test_programs/execution_success/unsafe_range_constraint/Nargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+name = "unsafe_range_constraint"
```

Comment:
Why is this unsafe? It's just an assertion.

Reply:
I named it unsafe because the user needs to remember that the field element fits within some amount of bits, ideally they use `as` if applicable. Can change it to `assert_max_bits` though